frontend-react/src/components/Table/Table.test.tsx

```diff
@@ -1,28 +1,48 @@
-import { fireEvent, screen } from "@testing-library/react";
+import { fireEvent, render, screen } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
+import { act } from "react-dom/test-utils";
 
 import { renderWithRouter } from "../../utils/CustomRenderUtils";
 import { mockFilterManager } from "../../hooks/filters/mocks/MockFilterManager";
 import { SortSettingsActionType } from "../../hooks/filters/UseSortOrder";
 
 import { TestTable } from "./TestTable";
-import Table, { ColumnConfig, TableConfig, TableRow } from "./Table";
+import Table, {
+    ColumnConfig,
+    TableConfig,
+    TableRow,
+    TableRows,
+    ColumnData,
+} from "./Table";
 
 /* Table generation tools */
 
-const getSetOfRows = (count: number) => {
+const getSetOfRows = (count: number, linkable: boolean = true) => {
```

Comment:
changes here are to allow for testing isolated functionality without wrapping in a router